HW-15

Vadoniy opened the pull request.

Commit:
HW-15

stvort commented on homework-15/new-year/src/main/java/ru/otus/flow/PreparationFlowConfiguration.java:
```diff
@@ -0,0 +1,89 @@
+    public IntegrationFlow christmasTreeFlow() {
+        return IntegrationFlows
+                .from("partyPreparationChannel")
+                .handle("luckService", "getLuck")
```
Во `flow`, `channel` и `handle` слева можно вписать переменную поля внедренного сервиса. Так сложнее сделать ошибку в строке с именем

stvort commented on homework-15/new-year/src/main/java/ru/otus/flow/PreparationFlowConfiguration.java:
```diff
@@ -0,0 +1,89 @@
+        return IntegrationFlows
+                .from("partyPreparationChannel")
+                .handle("luckService", "getLuck")
+                .split()
```
Что и на что мы тут делим и соответственно что потом агрегируем?

Commit:
HW-15 review fixes:
1) beans import instead of bean names string in flow configuration
2) removed excess split() and aggregate() methods, because they were not used

Commit:
HW-15 review fixes:
1) deleted split() method in kitchenFlow
2) channel names are replaced with beans
